rust_src/src/dispnew.rs

```diff
@@ -0,0 +1,42 @@
+//! Updating of data structures for redisplay.
+
+use remacs_macros::lisp_fn;
+use remacs_sys::{current_timespec, dtotimespec, timespec_add, timespec_sub,
+                 wait_reading_process_output, WAIT_READING_MAX};
+use lisp::LispObject;
+use std::{ptr, cmp};
+use floatfns::extract_float;
+
+
+/// Pause, without updating display, for SECONDS seconds.
+/// -SECONDS may be a floating-point value, meaning that you can wait for a
+/// -fraction of a second.  Optional second arg MILLISECONDS specifies an
+/// -additional wait period, in milliseconds; this is for backwards compatibility.
+/// -\(Not all operating systems support waiting for a fraction of a second.)
```

Comment:
These hyphens look like they've been copy-pasted by accident?

Also, you don't need to escape parens in Rust docstrings (because the paren is not the beginning of the line so Emacs doesn't get confused).